modules/apps/asset/asset-publisher-test/src/testIntegration/java/com/liferay/asset/publisher/util/test/AssetPublisherHelperTest.java

```diff
@@ -526,6 +532,30 @@ public void testGetAssetTagNamesNotContainsAnyTagNames() throws Exception {
 			Arrays.toString(assetTagNames), 0, assetTagNames.length);
 	}
 
+	@Test
+	public void testGetItemSelectorScopeGroupWithLayoutPrototype()
+		throws Exception {
+
+		LayoutPageTemplateEntry layoutPageTemplateEntry =
+			LayoutPageTemplateEntryServiceUtil.addLayoutPageTemplateEntry(
+				null, _group1.getGroupId(), 0, RandomTestUtil.randomString(),
+				LayoutPageTemplateEntryTypeConstants.WIDGET_PAGE, 0,
+				WorkflowConstants.STATUS_APPROVED,
+				ServiceContextTestUtil.getServiceContext(
+					_group1.getGroupId(), TestPropsValues.getUserId()));
+
+		LayoutPrototype layoutPrototype =
+			LayoutPrototypeLocalServiceUtil.getLayoutPrototype(
+				layoutPageTemplateEntry.getLayoutPrototypeId());
```

Comment:
Can you inject the services with an annotation instead of using the static Util classes?

Reply:
fixed both Util call.